cypress/integration/pages/onDemandTV/helpers.js

```diff
@@ -0,0 +1,43 @@
+import path from 'ramda/src/path';
+import envConfig from '../../../support/config/envs';
+
+// the externalId `bbc_oromo_radio` is overriden to `bbc_afaanoromoo` in production code
+const getBrandId = externalId =>
+  externalId.replace('bbc_oromo_radio', 'bbc_afaanoromoo_radio');
+
+const getServiceName = producerName =>
+  producerName.toLowerCase().replace('indonesian', 'indonesia');
+
+export const getEmbedUrl = (body, language) => {
+  const externalId = body.metadata.createdBy;
+  const brandId = getBrandId(externalId);
+  const producerName = body.metadata.analyticsLabels.producer;
+  const serviceName = getServiceName(producerName);
+  const { pid } = body.metadata.locators;
+
+  return [
+    envConfig.avEmbedBaseUrl,
+    'ws/av-embeds/media',
+    serviceName,
+    brandId,
+    pid,
+    language,
+  ].join('/');
+};
+
+export const isExpired = jsonData => {
+  const episodeAvailableUntil = path(
```

Comment:
Will we need to change this when we update the logic here for episodes that don't expire?

Reply:
I think so actually. I'll take a look at the other PR that is related to that and update this.

Reply:
Should we look to reusing the onDemandRadio helpers here instead? There's a lot of duplication and the pages / tests are very similar...